src/app/component/common/Tables/active-position-container.tsx

```diff
@@ -1,34 +1,41 @@
-'use client'
-import { FC, useState } from "react"
+"use client";
+import { FC, useState } from "react";
 import positionsInfo from "@/app/data/carddata";
 import ActivePositionsTable from "./active-positions-table";
 import { useTranslations } from "next-intl";
 
 const ActivePositions: FC = () => {
   const t = useTranslations("ActivePositions");
-    const [active, setActive] = useState(true);
-   let activePositionsTableData = active? positionsInfo.slice(0,2):positionsInfo
-    const handleClick = ()=>{
-        if(active){
-          setActive(false)
-        }else{
-          setActive(true)
-        }
-        
+  const [active, setActive] = useState(true);
+  let activePositionsTableData = active
+    ? positionsInfo.slice(0, 2)
+    : positionsInfo;
+  const handleClick = () => {
+    if (active) {
+      setActive(false);
+    } else {
+      setActive(true);
     }
```

Comment:
I think Eddy made this

`setActive(active);`

Reply:
yes